src/loadgenerator/loadgen.sh

```diff
@@ -32,4 +32,4 @@ if test $STATUSCODE -ne 200; then
 fi
 
 # else, run loadgen
-locust --host="http://${FRONTEND_ADDR}" --no-web -c "${USERS:-10}" 2>&1
```

Comment:
can we omit -r? It's not defined, so adding a default -r=10 potentially changes behavior (that's not something we want).

Reply:
Done